lib/administrate/base_dashboard.rb

```diff
@@ -74,5 +72,11 @@ def association_includes
     def attribute_not_found_message(attr)
       "Attribute #{attr} could not be found in #{self.class}::ATTRIBUTE_TYPES"
     end
+
+    def association_classes
+      @association_classes ||=
+        ObjectSpace.each_object(Class).
+          select { |klass| klass < Administrate::Field::Associative }
+    end
```

Comment:
Mmm, I like this, that's smart.

I'm not aware of anything where this might cause a problem; is there something you'd thought about?

Reply:
No, nothing that I can think of. I just noticed that suddenly a new item appeared in the list, and wondered if there could be implications, but couldn't think of any.

An alternative would be to keep a list, but provide an API for extension creators to add to it. It would be less developer-friendly though.